fiona/drvsupport.py

```diff
@@ -56,7 +56,7 @@
     # GeoJSON 	GeoJSON 	Yes 	Yes 	Yes
     ("GeoJSON", "raw"),
     # GeoJSONSeq 	GeoJSON sequences 	Yes 	Yes 	Yes 
-    ("GeoJSONSeq", "rw"),
+    ("GeoJSONSeq", "raw"),
```

Comment:
"raw" only since GDAL 3.6

Reply:
It is limited in the `driver_mode_mingdal` dict:  https://github.com/Toblerity/Fiona/blob/ac7aeeb47fd6db4733187c3117303a1bcbc4640a/fiona/drvsupport.py#L169